### PR TITLE
Update UG based on aileen comments

### PR DESCRIPTION
**Pull request by enhao25**
Those issues that are targeted are highlighted in yellow here https://docs.google.com/document/d/1qYt3xiYssBUyYmbzTGw3qRt7wIUUU71zJtv7alTxOdY/edit?usp=sharing.

Also added additional notes to delete_student and add_student section.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -404,9 +410,9 @@ before and after 2021-03-31 18:00
 
 #### 4.2.5 Fees
 
-##### Getting monthly fee for a particular student: `fee`
+##### Calculating monthly fee for a particular student: `fee`
 
-Gets the monthly fee for a particular student for a particular month and year.
+Calculates and retrieves the monthly fee for a particular student for a particular month and year.
```
Comment (JonahhGohh) [inline docs/UserGuide.md]: What do you think of rephrasing it to "Calculates and **displays**" ?
Feels to me that "retrieves" does not really explain it well enough here?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -237,15 +237,16 @@ Adds a student to TutorBuddy.
...
+:information_source: TutorBuddy automatically shows the full list of students and sessions after a successful <code>add_student</code> command.
 </div>
 
 Example:
 * `add_student n/John Doe p/98765432 e/johnd@example.com a/John street, Block 123, #01-01 l/Sec2 g/95421323 r/Mother`
 
 ##### Locating student by name: `find_student`
 
-Find student(s) whose names contain any of the given keywords.
+Filters the list to only include students, and their respective session whose student name contains any of the given keywords.
```
Comment (JonahhGohh) [inline docs/UserGuide.md]: On line 249,
"Filters the **student and session** list to only include **student(s)**, and their respective **session(s)** whose student name contains any of the given keywords."

**Review by JonahhGohh — COMMENTED** [review]
Thanks so much for updating this! 😄 
I just have some minor phrasing suggestions.

File: docs/UserGuide.md (hunk 2 above)
Comment (enhao25) [reply docs/UserGuide.md]: Right, this sounds better! Have updated it in the latest commit.

File: docs/UserGuide.md (hunk 1 above)
Comment (enhao25) [reply docs/UserGuide.md]: Ah right, now that I read again, retrieves doesn't sound right in this case, I have changed to "displays" in the latest commit

**Review by JonahhGohh — APPROVED** [review]
LGTM! 😺

File: docs/UserGuide.md [hunk 3]
```diff
@@ -417,9 +423,9 @@ before and after 2021-03-31 18:00
 
 #### 4.2.5 Fees
 
-##### Getting monthly fee for a particular student: `fee`
+##### Calculating monthly fee for a particular student: `fee`
```
Comment (samleewy) [inline docs/UserGuide.md:426]: Actually not sure if we should use 'calculating', maybe 'display'?

**Review by samleewy — COMMENTED** [review]
one minor comment for the fees part!

File: docs/UserGuide.md (hunk 3 above)
Comment (enhao25) [reply docs/UserGuide.md:426]: O.o I tink that aileen has mention that calculates in this case is better for the user to understand what the feature is doing. I will just keep it this way if you are ok with this.

**Pull request merged**